src/components/LoanFinding/PartnerSpotlightSection.vue

```diff
@@ -1,13 +1,15 @@
 <template>
-	<div>
-		<m-f-i-hero />
+	<div class="tw-w-full tw-bg-secondary">
```

Comment:
I was hoping we could pull these BGs from contentful, but I see that this actually isn't a contentful-driven page.

Reply:
yeah, we need to add some extra work to move this to contentful